File: datasets/IR-Plag/original-java-rest-kotlin/case-02/non-plagiarized/11/T02.kt
package IR

import java.text.DecimalFormat
import java.util.*

/**
 *
 * @author B00A50C448238A71ED479F81FA4D9066
 */

object T02 {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        val area: Double
        val volume: Double

        print("Enter the radius and length of a cylinder: ")
        val radius = sc.nextDouble()
        val length = sc.nextDouble()

        area = radius * radius * 3.14159
        volume = area * length

        val a = DecimalFormat("0.0000")
        val v = DecimalFormat("0.0")

        println("The area is " + a.format(area))
        println("The volume is " + v.format(volume))
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/non-plagiarized/11/T02.ts
import { Scanner } from 'readline-sync';
import { DecimalFormat } from 'decimal-format';

function main(): void {
    const sc = new Scanner();
    let radius: number, length: number, area: number, volume: number;

    console.log('Enter the radius and length of a cylinder: ');
    radius = sc.nextDouble();
    length = sc.nextDouble();

    area = radius * radius * 3.14159;
    volume = area * length;

    const a = new DecimalFormat('0.0000');
    const v = new DecimalFormat('0.0');

    console.log(`The area is ${a.format(area)}`);
    console.log(`The volume is ${v.format(volume)}`);
}

main();

